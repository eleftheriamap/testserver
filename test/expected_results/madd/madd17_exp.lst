
test/exp/test/gen/madd/madd17_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	d1592dad 	sub	x13, x13, #0x64b, lsl #12
   4:	d1477d29 	sub	x9, x9, #0x1df, lsl #12
   8:	11137dad 	add	w13, w13, #0x4df
   c:	9b0935b3 	madd	x19, x13, x9, x13
  10:	8a000000 	and	x0, x0, x0
