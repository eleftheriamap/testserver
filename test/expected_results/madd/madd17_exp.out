Registers:
X00    = 0000000000000000
X01    = 0000000000000000
X02    = 0000000000000000
X03    = 0000000000000000
X04    = 0000000000000000
X05    = 0000000000000000
X06    = 0000000000000000
X07    = 0000000000000000
X08    = 0000000000000000
X09    = ffffffffffe21000
X10    = 0000000000000000
X11    = 0000000000000000
X12    = 0000000000000000
X13    = 00000000ff9b54df
X14    = 0000000000000000
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = ffe21bc6c2c744df
X20    = 0000000000000000
X21    = 0000000000000000
X22    = 0000000000000000
X23    = 0000000000000000
X24    = 0000000000000000
X25    = 0000000000000000
X26    = 0000000000000000
X27    = 0000000000000000
X28    = 0000000000000000
X29    = 0000000000000000
X30    = 0000000000000000
PC     = 0000000000000010
PSTATE : -Z--
Non-zero memory:
0x00000000 : 0xd1592dad
0x00000004 : 0xd1477d29
0x00000008 : 0x11137dad
0x0000000c : 0x9b0935b3
0x00000010 : 0x8a000000
